.expeditor/verify.pipeline.yml

```diff
@@ -580,3 +580,33 @@ steps:
   expeditor:
     executor:
       docker:
+
+- label: ":habicat: Linux plan"
+  commands:
+    - sudo ./scripts/ci/install-hab.sh 'x86_64-linux'
+    - sudo ./scripts/ci/verify-plan.sh
+  expeditor:
+    executor:
+      linux:
+        privileged: true
+        single-use: true
+
+- label: ":habicat: Linux plan (kernel2)"
+  commands:
+    - sudo ./scripts/ci/install-hab.sh 'x86_64-linux-kernel2'
+    - sudo ./scripts/ci/verify-plan.sh
+  expeditor:
+    executor:
+      linux:
+        privileged: true
+        single-use: true
+
+- label: ":habicat: Windows plan"
+  commands:
+    - ./scripts/ci/verify-plan.ps1
```

Comment:
Could add a `./scripts/ci/install-hab.ps1` and extract the hab installation from `verify-plan.ps1` so this test steps looks like the other 2 Linux ones.

Reply:
The current approach in windows is slightly different from the `install-hab.sh` so I'd like to avoid changing it with this PR if that is ok.

I can certainly modify it in a future PR if you'd like.